docs/pages/management/admin/uninstall-teleport.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: Uninstall Teleport
+description: How to remove Teleport from your system
+---
+
+This guide explains how to uninstall Teleport binaries completely.
+
+## Prerequisites
+
+- A Node with Teleport installed.
+
+<Admonition type="warning">
+These instructions apply to bare-metal installations of Teleport only.
```

Comment:
I don't know if we should say _bare-metal_ here. That's an overloaded term and this usage of it is arguably inaccurate.